deepinv/transform/scale.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch.nn.functional as F
+from torch.nn import Module
+
+
+def sample_from(values, shape=(1,), dtype=torch.float32, device="cpu"):
+    """Sample a random tensor from a list of values"""
+    values = torch.tensor(values, device=device, dtype=dtype)
+    N = torch.tensor(len(values), device=device, dtype=dtype)
+    indices = torch.floor(N * torch.rand(shape, device=device, dtype=dtype)).to(
+        torch.int
+    )
+    return values[indices]
+
+
+class Scale(Module):
+    """
+    2D Scaling
```

Comment:
Could you make a short description of what "2D scaling" is, before adding ref to paper? Then you can also add pointers to this function in [https://deepinv.github.io/deepinv/deepinv.loss.html#transforms](https://deepinv.github.io/deepinv/deepinv.loss.html#transforms)? (i.e. add an entry in [here](https://github.com/deepinv/deepinv/blob/main/docs/source/deepinv.loss.rst))